polyphemus/lib/ipi/flow_wsp_file_linker.rb

```diff
@@ -0,0 +1,58 @@
+require 'etna'
+require 'json'
+
+class IpiFlowWspLinker < Etna::Clients::Magma::FileLinkingWorkflow
+  def initialize(**opts)
+    super(**{bucket_name: 'integral_data', model_name: 'patient'}.update(opts))
+  end
+
+  def find_matches
+    {}.tap do |all_matches|
+      metis_client.find(
+        Etna::Clients::Metis::FindRequest.new(
+          project_name: project_name,
+          bucket_name: bucket_name,
+          params: [Etna::Clients::Metis::FindParam.new(
```

Comment:
Oh nice, using the metis command makes this much faster then!

Reply:
I was thinking in the near future we could add "cursors" (last updated at scanned) to make all these etl-like jobs run faster, too.  They'd only have to scan files that have updated since the last attempt.

Reply:
Oh yeah, that would be great!